### ISSUE TITLE
Consider adding a timelock policy

### ISSUE THREAD

**Issue by landabaso** [LedgerHQ/app-bitcoin-new#45]
I've developed a vault-wallet (it will published/opensourced soon) that is able to spend locked utxos such as this one:

```
<KEY_1> OP_CHECKSIG OP_NOTIF
  <KEY_2> OP_CHECKSIG
OP_ELSE
  <LOCK_TIME> OP_CHECKSEQUENCEVERIFY
OP_ENDIF
```

I have successfully integrated the wallet with Ledger devices using with the old btc app using `signP2SHTransaction`.

However, it is currently not possible to use the new app with such script now.

I understand you'll allow all sort of policies in the future but it looks like they're currently restricted to more battle-tested: P2PKH, P2WPKH, P2SH-P2WPKH, TR, and multisig.

I would like to ask you to include this policy in the collection of initial supported policies so that I can start preparing the integration with the new application some time in advance:

`wsh(or(and(pk(KEY_1),older(TIME)),pk(LOCK_TIME)))`

**Comment by bigspider** [comment]
No promises on the precise ETA but I expect a dev-release of miniscript support by this summer, hopefully by July.
I'll let you know once I have something stable enough to play with!

**Comment by bigspider** [comment]
Closing this, as it's already included in #44 which is in the works right now.
